sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/component.py

```diff
@@ -423,7 +423,7 @@ def _from_container_rest_object(cls, component_container_rest_object: ComponentC
             properties=component_container_details.properties,
             type=NodeType._CONTAINER,
             # Set this field to None as it hold a default True in init.
-            is_deterministic=None,
+            is_deterministic=None,  # type: ignore[arg-type]
```

Comment:
Seems like this one needs more clarity from the responsible team. Not a blocker, but a follow up to figure out what this should be set to by default (False vs. True). I'll reach out to the Component owner and make a work item for them to update this and remove the type: ignore comment.